docs/templates/icons.md

```diff
@@ -0,0 +1,20 @@
+# Icons
+
+---
+
+Coder uses icons in several places, including ones that can be configured
+throughout the app, or specified in your Terraform. They're specified by a URL,
+which can be to an image hosted on a CDN of your own, or one of the icons that
+come included with your Coder deployment.
```

Comment:
```suggestion
In Coder, icons are utilized in various areas, and can be customized to enhance visual recognition and representation. They are defined via a URL, which can either be an image hosted on your CDN or one of the icons included with your Coder deployment.

1. **Template Icons**:
   - Customize icons to differentiate templates within the application.

2. **Terraform Resources**:
   - [`coder_parameter`](https://registry.terraform.io/providers/coder/coder/latest/docs/data-sources/parameter#icon): Specify an icon by providing a URL in the `icon` field.
   - [`coder_app`](https://registry.terraform.io/providers/coder/coder/latest/docs/resources/app#icon): Assign an icon by setting the `icon` field with a URL.
   - [`coder_script`](https://registry.terraform.io/providers/coder/coder/latest/docs/resources/script#icon): Define an icon by inserting a URL in the `icon` field.

3. **External Auth Providers**:
   - Set custom icons for [External Auth Providers](https://coder.com/docs/v2/latest/admin/external-auth) to visually distinguish between different authentication methods.

4. **Icon Sources**:
   - **CDNs**: Use any icon URL from a CDN accessible from your deployment. e.g., https://upload.wikimedia.org/wikipedia/commons/9/93/Amazon_Web_Services_Logo.svg
   - **Bundled Icons**: Use icons included with your Coder deployment by providing their path. e.g., "/icon/aws.svg"
```

Reply:
I didn't know all the places it was used, so this is helpful!